.gitignore

```diff
@@ -1,11 +1,5 @@
 result*
-*.o
-*.a
-/*.exe
-
-.dep**
-.target.*
-.lock.*
+/build
```

Comment:
Good news, even this single line is unnecessary: https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

:)

Reply:
Removed, thanks!